### PR TITLE
Fix totally failed API

### PR DESCRIPTION
**Pull request by greenEkatherine**
Related to #78

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -29,14 +29,16 @@ jobs:
...
             # to find out other devices run "avdmanager list device"
-            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 9 --force
+            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 17 --force
 
             $ANDROID_HOME/emulator/emulator -list-avds
 
             echo "Starting emulator"
-            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -skin 1080x1920 -no-snapshot > /dev/null 2>&1 &
+            # increase memory size to 2048M to avoid error "System UI is not responding"
+            sed -i 's/^\(hw.ramSize=\).*$/\12048M/' ~/.android/avd/xamarin_android_emulator.avd/config.ini
```
Comment (omen273) [inline azure-pipelines.yml:40]: What does \1 mean?

**Review by omen273 — CHANGES_REQUESTED** [review]
Please address my comments

File: azure-pipelines.yml (hunk 1 above)
Comment (omen273) [reply azure-pipelines.yml:40]: And also please add comment about it

Comment (greenEkatherine) [reply azure-pipelines.yml:40]: It means to keep the part that is in braces in the left-side expression. So I will get `hw.ramSize=96M` -> `hw.ramSize=2048M`. Without it I will get `hw.ramSize=96M` -> `2048M`

**Review by omen273 — APPROVED** [review]

**Comment by greenEkatherine** [comment]
ready to merge, @omen273 please a have a look

**Review by omen273 — APPROVED** [review]

**Pull request merged**